### PR TITLE
Update

### PR DESCRIPTION
**Pull request by andreiavrammsd**
DB volumes, ports, management.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,35 +44,59 @@ services:
...
-      MYSQL_RANDOM_ROOT_PASSWORD: yes # <not set>
+      MYSQL_ALLOW_EMPTY_PASSWORD: 'no'
+      MYSQL_RANDOM_ROOT_PASSWORD: 'no'
+    volumes:
+      - mariadb:/var/lib/mysql
+  phpmyadmin:
+    image: phpmyadmin/phpmyadmin
+    restart: unless-stopped
+    ports:
+      - "8080:80"
```
Comment (cristaloleg) [inline README.md:63]: huh, looks problematic, but I hope everyone will see port 80 and will change it

**Review by cristaloleg — COMMENTED** [review]
Thanks! Great update 👍

**Review by cristaloleg — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (andreiavrammsd) [reply README.md:63]: 80 is inside the container. Do you mean 8080 being too used for other apps?

Comment (cristaloleg) [reply README.md:63]: Ah, yeah, 8080, but never mind.

**Pull request merged**